### PR TITLE
Added AlexaSkillsKit and updated the section listings for Machine Lea…

### PR DESCRIPTION
**Pull request by yoha**
Same as title.

## Project URL

https://github.com/choefele/AlexaSkillsKit

## Description

1. Per our discussion earlier about creating a new section for personal assistant libraries, I have implemented the changes as followed:

# Machine Learning
## Artificial Intelligence
- Swift-AI
- Swift-Brain
- AIToolbox

## Personal Assistant
- AlexaSkillsKit

## Miscellaneous
- Tensorflow-iOS

2.  I added AlexaSkillsKit under PA subcategory of ML. 
 
## Why it should be included to `awesome-ios` (optional)
It's pretty cool lib for those who wants to hook into amazon's PA API. 

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 or later
- [x] Supports Swift 3
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 2 link issues</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
186 | [502](https://httpstatuses.com/502) | https://onliquid.com 
424 | [404](https://httpstatuses.com/404) | https://github.com/faunadb/faunadb-swift 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
     - [Logging](#logging)
     - [Localization](#localization)
     - [Machine Learning](#machine-learning)
+        - [Artificial Intelligence](#artificial-intelligence)
+        - [Personal Assistant](#personal-assistant)
+        - [Miscellaneous](#miscellaneous)
```
Comment (dkhamsing) [inline README.md]: Because of the way these anchor works, it should be Miscellaneous Machine Learning

File: README.md [hunk 2]
```diff
@@ -654,9 +657,15 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
...
 ## Machine Learning
+### Artificial Intelligence
 * [Swift-AI](https://github.com/collinhundley/Swift-AI) - Highly optimized Artificial Intelligence and Machine Learning library written in Swift. :large_orange_diamond:
 * [Swift-Brain](https://github.com/vlall/Swift-Brain) - Artificial Intelligence/Machine Learning data structures and Swift algorithms for future iOS development. Bayes theorem, Neural Networks, and more AI. :large_orange_diamond:
 * [AIToolbox](https://github.com/KevinCoble/AIToolbox) - A toolbox of AI modules written in Swift: Graphs/Trees, Linear Regression, Support Vector Machines, Neural Networks, PCA, KMeans, Genetic Algorithms, MDP, Mixture of Gaussians. :large_orange_diamond:
+
+### Personal Assistant
+* [AlexaSkillsKit](https://github.com/choefele/AlexaSkillsKit) - Swift library to develop custom Alexa Skills. :large_orange_diamond:
+
+### Miscellaneous
```
Comment (dkhamsing) [inline README.md]: Same

File: README.md (hunk 1 above)
Comment (yoha) [inline README.md]: You mean the following
- [Miscellaneous](#miscellaneous)
should be changed to 
- [Miscellaneous](#miscellaneous-machine-learning)
?

Can you elaborate?

File: README.md (hunk 2 above)
Comment (dkhamsing) [inline README.md]: yes

**Pull request closed without merging**